tools/lkl/lib/hijack/hijack.c

```diff
@@ -160,6 +160,19 @@ HOOK_FD_CALL(splice)
 HOOK_FD_CALL(vmsplice)
 HOOK_CALL_USE_HOST_BEFORE_START(pipe);
 
+HOOK_CALL_USE_HOST_BEFORE_START(accept4);
+int accept4(int fd, struct sockaddr* addr, socklen_t* addrlen, int flags)
```

Comment:
Please use `struct sockaddr *addr` instead of `struct sockaddr* addr` to stay consistent with the Linux kernel coding style.